evalml/tests/model_understanding_tests/test_partial_dependence.py

```diff
@@ -516,17 +516,17 @@ def round_dict_keys(dictionary, places=6):
     assert num_cat_features == 73
 
     part_dep_ans = {
-        0.05824028901694482: 63,
-        0.1349235160940143: 1,
-        0.6353030372157324: 1,
-        0.031171284274810262: 1,
-        0.009093086236362272: 1,
-        0.33547688991040336: 1,
-        0.01746660818843149: 1,
-        0.018205973481273202: 1,
-        0.2876661156872482: 1,
-        0.015320197702897345: 1,
-        0.2821023107719306: 1,
+        0.08407392065748104: 63,
```

Comment:
Why did this change?

Reply:
the fraud test data set contains `Categoricals` that are being ignored now.